### ISSUE TITLE
Fix error to make updating dependency JSS possible.

### ISSUE THREAD

**Issue by vaalllaa** [thenativeweb/thenativeweb-ux#210]
### What is this bug about?
Right now, because of an error, an update for JSS from 10.0.2 to 10.0.3 is not possible

### What is the expected result?
All checks to have passed.

### What is the actual result?
Error is blocking from merging.

### What steps are needed to reproduce the bug?
-   thenativeweb/thenativeweb-ux#200

**Comment by vaalllaa** [comment]
This is what I came across so far: https://github.com/cssinjs/jss/issues/1203

**Comment by vaalllaa** [comment]
What I came across so far: https://github.com/cssinjs/jss/issues/1203

**Comment by vaalllaa** [comment]
@mattwagl could you please have a look at this issue. I tried, but still not sure what exactly I have to adjust to fit the updated usage.

**Comment by mattwagl** [comment]
The typings for `createUseStyles` have been changed. This causes several issues. I created a [CodeSandbox](https://codesandbox.io/s/romantic-breeze-ocnbo) to do several smaller tests in isolation that helped to further investigate the issue. But I first want to tackle this issue when the styleguide branch has been merged as it touches a lot of the components. Then I'll try to update the ux package to JSS 1.0.3 until then we'll have to wait with this issue.
